### PR TITLE
Text content

### PR DESCRIPTION
**Pull request by keiffer01**
Just added text descriptions to each section (the About Me and each item of the portfolio, as well as actual fun facts). No other visual changes done in this branch.

TESTING: none

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -37,13 +37,16 @@ <h5 id="blurb">(that's pronounced "key-fur"!)</h5>
...
-          <li class="link"><a href="#" target="_blank"><ion-icon name="logo-linkedin"></ion-icon></ion-icon></ion-icon></a></li>
-          <li class="link"><a href="#" target="_blank"><ion-icon name="logo-octocat"></ion-icon></ion-icon></ion-icon></a></li>
+          <li class="link"><a href="mailto:keiffer.acoba@gmail.com" target="_blank"><ion-icon name="mail-outline"></ion-icon></ion-icon></a></li>
+          <li class="link"><a href="https://www.facebook.com/keiffer.acoba" target="_blank"><ion-icon name="logo-facebook"></ion-icon></ion-icon></ion-icon></a></li>
+          <li class="link"><a href="https://instagram.com/keifferacoba" target="_blank"><ion-icon name="logo-instagram"></ion-icon></ion-icon></ion-icon></a></li>
+          <li class="link"><a href="https://www.linkedin.com/in/keiffer-acoba-90426416b" target="_blank"><ion-icon name="logo-linkedin"></ion-icon></ion-icon></ion-icon></a></li>
+          <li class="link"><a href="https://github.com/keiffer01" target="_blank"><ion-icon name="logo-octocat"></ion-icon></ion-icon></ion-icon></a></li>
```
Comment (beiebi0) [inline portfolio/src/main/webapp/index.html]: Are those extra "<\/ion-icon>" necessary? Could you please verify?

Wondering if "<ion-icon name="mail-outline" \/> " works in this case if nothing added in betwee.

File: portfolio/src/main/webapp/portfolio.html [hunk 2]
```diff
@@ -27,7 +27,8 @@
       </div>
       <div class="column right">
         <h3 class="header">This Website</h3>
-        <p>Lorem ipsum blah blah put info hereeeeeeee</p>
+        <p>This website was created as part of training during Google's 2020 STEP Summer internship. It was created completely from scratch without the use of any frameworks. Currently, I'd still consider it a work-in-progress and will likely continue to add to and update it throughout the future.</p>
+        </p>Checkout the project's <a href="https://github.com/keiffer01/step">Github repo</a>.
```
Comment (beiebi0) [inline portfolio/src/main/webapp/portfolio.html]: extra <\/p>?

**Review by beiebi0 — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (HiramSilvey) [reply portfolio/src/main/webapp/index.html]: Which quotes are you referring to?

File: portfolio/src/main/webapp/portfolio.html [hunk 3]
```diff
@@ -38,7 +39,8 @@ <h3 class="header">This Website</h3>
       </div>
       <div class="column right">
         <h3 class="header">PointMe</h3>
-        <p>Lorem ipsum blah blah put info hereeeeeeee</p>
+        <p>PointMe is a project created by a team of 4 in the HackCMU 2019 hackathon, winning 2nd place overall. Having suffered with the workings of managing memory in C and growing frustrated with the cryptic messages being returned by valgrind, we wrote PointMe as a visual debugger to show you where on the stack and the heap variables were located, as well as which pointers referenced what. I contributed to part of the project that involved parsing the contents of the .c file before passing them on to the frontend.</p>
+        <p>Checkout more info the project's <a href="https://devpost.com/software/pointme-7im4b0">Devpost listing</a>.
```
Comment (HiramSilvey) [inline portfolio/src/main/webapp/portfolio.html]: Missing closing p tag?

**Review by HiramSilvey — CHANGES_REQUESTED** [review]
Not required, but it would be nice if you shortened each line to <=80 chars in length. That would make it more readable in both text editors as well as on GitHub. Depending on the editor you use, there is likely a plugin to help accomplish this task. I can help you track one down if you're interested.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (beiebi0) [reply portfolio/src/main/webapp/index.html]: aha I forgot I am writing HTML. it doesn't show "<\/ion-icon>"  just updated the comment above.

File: portfolio/src/main/webapp/portfolio.html (hunk 2 above)
Comment (keiffer01) [reply portfolio/src/main/webapp/portfolio.html]: Whoops, overlooked that, added.

File: portfolio/src/main/webapp/portfolio.html (hunk 3 above)
Comment (keiffer01) [reply portfolio/src/main/webapp/portfolio.html]: Yeah overlooked some tags by accident, added now

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (HiramSilvey) [reply portfolio/src/main/webapp/index.html]: According to the usage documentation, it looks like you need the opening and closing tags separate: https://ionicons.com/usage/

**Review by HiramSilvey — APPROVED** [review]

**Review by beiebi0 — APPROVED** [review]

**Pull request merged**